integration/control_plane.go

```diff
@@ -10,8 +10,9 @@ import (
 // Right now, that means Etcd and your APIServer. This is likely to increase in
```

Comment:
This comment is out of date

Reply:
Thanks. I will recheck on the docs and typos overall.